CONTRIBUTING.md

```diff
@@ -7,11 +7,10 @@
 - Create a feature branch (git checkout -b your-new-feature)
 
 ### Step 3
-- Commit the changes (git commit -am 'Add some feature') (will additional add the new changes)
+- Commit the changes (git commit -m 'Add some feature') (will additionally add the new changes)
```

Comment:
Need the `a` too otherwise changes won't be added (as the bit in brackets suggests).